superdesk/default_settings.py

```diff
@@ -446,6 +455,9 @@ def env(variable, fallback_value=None):
 #: The number of minutes before published content items are purged
 PUBLISHED_CONTENT_EXPIRY_MINUTES = int(env('PUBLISHED_CONTENT_EXPIRY_MINUTES', 0))
 
+#: The number of minutes before audit content is purged
+AUDIT_EXPIRY_MINUTES = int(env('AUDIT_EXPIRY_MINUTES', 0))
```

Comment:
maybe this could be enabled by default, 1 day or 7 days, not sure how big it gets?